Adding the namespace to the ExamplesHubHomeButtonActivation script

cre8ivepark opened the pull request:
## Overview
- Adding the namespace Microsoft.MixedReality.Toolkit.UI to the example script 
- Moved the script ExamplesHubHomeButtonActivation to SDK folder since it is used by ToggleFeaturesPanel prefab. 

This script is used to show or hide the home button in the ToggleFeaturesPanel.
![2019-09-20 23_10_07-Unity 2018 4 6f1 Personal - MRTKExamplesHub unity - MRTK-Public-Yoon - Universal](https://user-images.githubusercontent.com/13754172/65368944-eb273400-dbfb-11e9-9da6-d31e35ecd231.png)

julenka commented on Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Utilities/ExamplesHubHomeButtonActivation.cs:
```diff
@@ -0,0 +1,33 @@
+
+        private void OnEnable()
+        {
+            if (MixedRealityToolkit.IsSceneSystemEnabled == true)
```
This seems much more like an example to me -- the scrip doesn't have tons of value beyond the example. Additionally, the ToggleFeaturesPanel prefab is used in HandInteractionExamples, isnt' it? In that case we don't want a home button. It feels like toggle features with a home button should be a separate prefab, that lives in the examples folder. Then you could move this script back to Examples folder, where I think it belongs.

cre8ivepark replied:
Right. Tried to think about separating the home button from the ToggleFeaturesPanel or having duplicated ToggleFeaturesPanel for the Examples Hub. The challenge was, there are many example scenes that are using ToggleFeaturesPanel. I wanted to make the example scene work for both as a standalone and as part of the Example Hub.

cre8ivepark replied:
Putting a separate home button for the hub is easy but make it nicely aligned with ToggleFeaturesPanel would be a challenge.

cre8ivepark replied:
I'll duplicate the ToggleFeaturesPanel for the example hub specific version. I'll put this info in the README file. Thanks!

cre8ivepark commented:
- Created ToggleFeaturesPanelExamplesHub.prefab (with Home Button)
- Reverted the original ToggleFeaturesPanel.prefab 
- Removed script from both prefabs. (no longer needed to show or hide home button)
- Updated documentation with instruction to use 'ToggleFeaturesPanelExamplesHub'
- Updated Examples Hub button's interactable profile asset to resolve a warning about state mismatch. Assigned proper profile.

julenka commented:
/azp run
